docs/ref/settings.txt

```diff
@@ -393,6 +393,17 @@ Whether to use a secure cookie for the CSRF cookie. If this is set to ``True``,
 the cookie will be marked as "secure," which means browsers may ensure that the
 cookie is only sent under an HTTPS connection.
 
+.. setting:: CSRF_HEADER_NAME
+
+CSRF_HEADER_NAME
+------------------
+
+.. versionadded:: 1.9
+
+Default: ``HTTP_X_CSRFTOKEN``
```

Comment:
Don't remove the backticks -- just add single quotes -- like the default for FILE_CHARSET.